app/views/diabetes-content/health-problems-caused-by-diabetes-type-2.html

```diff
@@ -9,69 +9,89 @@
+- numbness
+- pain or tingling
+- problems with sex
+- constipation or diarrhoea
 
-###Watching your feet###
+Early treatment and good blood sugar control can prevent nerve damage getting worse. 
 
-You should look after your feet and check them every day when you have diabetes.  
+##Looking after your feet##
 
-Diabetes can reduce the blood supply to your feet cause a loss of feeling.  This means foot injuries don’t heal well and you may not notice if your foot is sore or injured. 
+Because of your diabetes, you should check your feet every day.  
 
-Simple things are important like:
-keeping feet clean and dry to avoid infection
-trying not to go barefoot outside to avoid niks and cuts
+Diabetes can reduce the blood supply to your feet and cause a loss of feeling.  This means:
 
-Speak to your GP or care team if you notice any changes, including:
+- foot injuries don’t heal well
+- you may not notice if your foot is sore or injured 
+
+Simple things are important, like:
+
+- keeping feet clean and dry to avoid infection
+- trying not to go barefoot outside to avoid niks and cuts
+- wearing shoes that fit well 
+
+Speak to your GP or care team if you notice any changes in your feet, including:
 
 - cuts, cracks or blisters
-- pain
-- itching
-- Numbness of the feet-
+- pain or tingling
+- numb feet
 
 Diabetes UK has advice on [how to check your feet](https://www.diabetes.org.uk/Guide-to-diabetes/Complications/Feet/)
 
-Your feet should also be checked by a foot specialist (podiatrist) every year.
+Your feet should also be checked every year by a foot specialist (podiatrist) or at your annual diabetes review.  
+
+[Book your diabetes foot check](http://nhsalpha.herokuapp.com/booking-with-context/going-for-regular-diabetes-check-ups)
   
 Sores or infections that aren’t treated early can lead to gangrene. More than 135 amputations are carried out every week in the UK due to diabetes.
 
-###Going for eye screening###
+##Checking your eyes##
+
+[Your eyes should be checked every year](http://www.nhs.uk/conditions/diabetes/pages/diabetic-eye-screening.aspx) for damaged blood vessels, which can cause sight problems [(diabetic retinopathy)](http://www.nhs.uk/Conditions/Diabetic-retinopathy/Pages/Introduction.aspx) and blindness.  
+
+Diabetes eye checks can detect damage before it affects your sight. Treating damaged blood vessels early can prevent sight problems. 
 
-[Your eyes should be checked every year] (http://www.nhs.uk/conditions/diabetes/pages/diabetic-eye-screening.aspx) for damaged blood vessels, which can cause sight problems [(diabetic retinopathy)] (http://www.nhs.uk/Conditions/Diabetic-retinopathy/Pages/Introduction.aspx) and blindness.  Eye checks can detect damage before you notice changes to your sight. Treating damaged blood vessels early may stop them affecting your sight. 
+[Book your diabetes eye check](http://nhsalpha.herokuapp.com/)
```

Comment:
Just noticed that this isn't the right link to book a service - it should be linking to `/book-an-appointment/diabetes-eye-screening/find-new-appointment`.

For simplicity I'm happy to merge this but @kimbod can you update the link in the Google Drive version and create a new pull request please?